tests/integration/components/lt-body-test.js

```diff
@@ -148,16 +148,20 @@ test('row expansion - multiple', function(assert) {
 });
 
 test('row actions', function(assert) {
```

Comment:
```js
test('row actions', async function(assert) {
```

By turning this into an [`async` function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function), you can use the `await` keyboard in the function body. [ember-native-dom-helpers](https://github.com/cibernox/ember-native-dom-helpers#acceptance-tests) returns a promise from all the interaction helpers, that resolves when the world has settled and there are no pending renders, requests or updates.